apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -234,6 +234,17 @@ class AppStore extends EventEmitter {
         this.emit('scheduleNotesChange');
     }
 
+    redoAction() {
```

Comment:
You missed the call to `autoSaveSchedule`.

Reply:
Accordingly, the `UndoAction` should be used but renamed to something like `UndoRedoAction` (or a better name that's not coming to me right now).